### PR TITLE
CLUS-firepower-restapi112-md1

### PR DESCRIPTION
**Pull request by ReneeBornstein**
* changed name of lab session to
   "this".
* added to sentences.
* added commas and colons

### CODE REVIEW

File: labs/firepower-restapi-112/1.md [hunk 1]
```diff
@@ -1,23 +1,20 @@
 # FMC - REST - API - 112
 
-## Lab Objective
+## Lab objective
```
Comment (paulzim) [inline labs/firepower-restapi-112/1.md]: Should just be "Objectives"

File: labs/firepower-restapi-112/1.md [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
 
-Basic understanding of Firepower Management Center (FMC) REST based APIs CRUD operations   
+This lab teaches you the basic understanding of Firepower Management Center (FMC) REST based APIs CRUD operations.   
 
-Completion time: 25 minutes
 
 ## Prerequisites
-Basic understanding of REST principles, firewalls
+It's best if you already have basic understanding of REST principles and firewalls.
```
Comment (paulzim) [inline labs/firepower-restapi-112/1.md]: "It's best..." is a judgement call. The original is better.

**Review by paulzim — CHANGES_REQUESTED** [review]

File: labs/firepower-restapi-112/1.md [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
-Completion time: 25 minutes
 
 ## Prerequisites
-Basic understanding of REST principles, firewalls
+Basic understanding of REST principles and firewalls.
 
 
-## Learning Objective
-After completing this module you will have a basic understanding of FMC REST APIs CRUD operations
+## Learning objective
```
Comment (annegentle) [inline labs/firepower-restapi-112/1.md]: You can delete "Learning objective" heading.

File: labs/firepower-restapi-112/1.md [hunk 4]
```diff
@@ -1,23 +1,20 @@
...
+After completing this module you will have a basic understanding of FMC REST APIs CRUD operations.
 
 
 ## Read operation GET
-If you are using DevNet sandbox  https://fmcrestapisandbox.cisco.com/api/api-explorer/
-
-For the CLUS 17 workshop session the username and password is provided to you. These credentials are only valid for the CLUS17 session.
+If you are using the [DevNet Sandbox] (https://fmcrestapisandbox.cisco.com/api/api-explorer/), for this workshop session the username and password is provided to you. These credentials are only valid for this session.
```
Comment (annegentle) [inline labs/firepower-restapi-112/1.md]: Having a space after the square bracket `]` means the link will not render correctly. I wonder also why this link is the fmcrestapisandbox, but in the prior PR the link was https://cisco-tid.cisco.com/api/api-explorer/?

File: labs/firepower-restapi-112/1.md [hunk 5]
```diff
@@ -37,21 +34,25 @@ The API Explorer URL is publically available. However, in order to login to the
...
+
+Since this is a READ operation you can press *GET* on the *API CONSOLE* pane to get all of the network objects from FMC.
 
 ![Figure: FMC API Explorer UI](/posts/files/firepower-restapi-112/assets/images/pic2.PNG)
 
+
 ## Write operation POST
 
-This time you need to click on the POST operation and Click Examples to get JSON
+This time you need to click on the POST operation, and Click Examples to get JSON.
```
Comment (annegentle) [inline labs/firepower-restapi-112/1.md]: Click Examples probably doesn't need to be capitalized.

File: labs/firepower-restapi-112/1.md [hunk 6]
```diff
@@ -67,7 +68,8 @@ Alternatively, You can copy JSON from here.
 ```
 
 
-Copy and paste the JSON in the API CONSOLE *Body* section and Click POST
+
+Copy and paste the JSON in the API CONSOLE *Body* section, and Click POST.
```
Comment (annegentle) [inline labs/firepower-restapi-112/1.md]: Click does not need to be capitalized here.

**Review by annegentle — CHANGES_REQUESTED** [review]
You could also delete the extra /posts/files/lab-id image references here if you want.

**Comment by ReneeBornstein** [comment]
Yet another one:

OMG, another weird error. I opened the GitHub doc link you sent, clicked on Files changed, and copied that to another window, so I could edit it. Usually I then just click on the pencil icon to edit the file. But this time I got this error:

[/var/folders/9x/dk6rpbtd6tnc0tqvtk73lz100000gn/T/com.microsoft.Outlook/WebArchiveCopyPasteTempFiles/cidimage001.png@01D40BD1.FE73A850]

I’ve never had that happen before. I am logged in, so I guess somehow my push access (whatever that is) suddenly isn’t working?

**Comment by ReneeBornstein** [comment]
Hmmm, so with all of these do you think I should have to reclone them, do the changes from your email on them, and send them back to you that way? And if so, when do I move them back to my Documents -> GitHub?

Or is this error something different?

**Comment by ReneeBornstein** [comment]
Made the changes in Atom.

**Review by annegentle — APPROVED** [review]

**Pull request merged**